var/spack/repos/builtin/packages/scine-qcmaquis/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class ScineQcmaquis(CMakePackage):
+    """Scine QCMaquis DMRG Solver"""
+
+    homepage = "https://scine.ethz.ch/download/qcmaquis"
+    git = "https://github.com/qcscine/qcmaquis.git"
+
+    maintainers("adam-grofe")
+
+    version("master", branch="master")
+    version("3.1.3", branch="release-3.1.3")
+    version("3.1.2", branch="release-3.1.2")
+    variant(
+        "blas",
+        values=("openblas", "mkl"),
+        default="openblas",
+        description="Which blas library to use.",
+    )
+    variant(
+        "symmetries",
+        default="TwoU1;SU2U1;SU2U1PG;TwoU1PG",
+        description='Wave functions symmetries to compile (e.g. "TwoU1;SU2U1;SU2U1PG;TwoU1PG")',
+    )
```

Comment:
Would it make sense for this package to make this a multi-valued variant that can take multiple values?  (See the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#multi-valued-variants.)

Reply:
I changed this variant to use the multi.